web/source/osk/visualKeyboard.ts

```diff
@@ -225,7 +225,10 @@ namespace com.keyman.osk {
       var width: number = OSKKey.getTextWidth(osk, keyText, styleSpec);
       if(width == 0 && keyText != '' && keyText != '\xa0') {
         // Add the Unicode 'empty circle' as a base support for needy diacritics.
-        keyText = '\u25cc' + keyText;
+
+        // Disabled by mcdurdin 2020-04-09; dotted circle display is inconsistent on iOS/Safari
```

Comment:
```suggestion
       // Disabled by mcdurdin 2020-10-19; dotted circle display is inconsistent on iOS/Safari
```

May also want to document that this should now be managed through custom fonts instead of handled dynamically within the engine.